### PR TITLE
Initial commit of Pleasuredome.xml

### PR DESCRIPTION
**Pull request by kerframil**
This ruleset covers the Pleasuredome Tracker and its associated forums.

### CODE REVIEW

File: src/chrome/content/rules/Pleasuredome.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+	<target host="www.pleasuredome.org.uk" />
+	<target host="forum.pleasuredome.org.uk" />
+
+	<test url="http://pleasuredome.org.uk/" />
+	<test url="http://www.pleasuredome.org.uk/" />
+	<test url="http://forums.pleasuredome.org.uk/" />
+
+	<securecookie host=".+" name=".+" />
+
+	<rule from="^https?://pleasuredome\."
```
Comment (J0WI) [inline src/chrome/content/rules/Pleasuredome.xml]: We don't redirect https domains.

File: src/chrome/content/rules/Pleasuredome.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+	      to="https://www.pleasuredome." />
```
Comment (J0WI) [inline src/chrome/content/rules/Pleasuredome.xml]: This is not a valid TLD

**Comment by kerframil** [comment]
Sorry, I didn't realise that self-signed certs are disallowed. Several parties have tried - and failed - to convince the webmaster to make use of a properly signed certificate. Therefore, this ruleset will remain inadmissible and I am closing the request.

**Comment by J0WI** [comment]
They are not disallowed, but then the rule should be `default_off`.

**Comment by kerframil** [comment]
Incidentally, it is neither mentioned [here](https://www.eff.org/https-everywhere/rulesets) nor [here](https://github.com/EFForg/https-everywhere/blob/master/ruleset-style.md) that redirects from https URLs are a QA violation. Further, some existing rulesets exploit the fact that it can be done.

```
$ cat *.xml | perl -ne 'print if /rule from="\^?https/ and not /<!--\s*<?rule/' | wc -l
98
```

Why does HTTPS-Everywhere honour such rules (downgrade attribute notwithstanding) if it is not considered to be an acceptable practice?

**Comment by J0WI** [comment]
The documentation will soon be improved: https://github.com/EFForg/https-everywhere/pull/8193/files#diff-6a3371457528722a734f3c51d9238c13R230

We hope to get rid of the existing cases, but sadly some servers are really bad configured (see also discussion started in #7717).

**Pull request merged**